### PR TITLE
Feature/285 popover component

### PR DESCRIPTION
**Pull request by anna-lach**

### CODE REVIEW

File: packages/components/inline-message/custom-elements.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "schemaVersion": "1.0.0",
+  "readme": "",
+  "modules": []
+}
```
[the remark this answers is not in the document]
Comment (anna-lach) [reply packages/components/inline-message/custom-elements.json:5]: right 😅

File: packages/components/popover/src/popover.ts [hunk 2]
```diff
@@ -45,13 +61,35 @@ export class Popover extends LitElement {
...
         </svg>
       </div>
     `;
   }
+
+  #onKeydown(event: KeyboardEvent): void {
+    if (event.code === 'Escape') {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      (this as HTMLElement).hidePopover();
```
Comment (jpzwarte) [inline packages/components/popover/src/popover.ts:92]: This is weird; `LitElement` already extends `HTMLElement`. This shouldn't be necessary?!

**Review by jpzwarte — APPROVED** [review]

File: packages/components/popover/src/popover.stories.ts [hunk 3]
```diff
@@ -21,81 +21,138 @@ export default {
...
+export const Dialog: Story = {
+  render: () => {
+    const onClick = (event: Event & { target: Button }): void => {
+      const popover = event.target.nextElementSibling as HTMLElement;
+
+      popover.togglePopover();
+    };
+
+    return html`
+      <!--added this to get the polyfill working in FF and Safari-->
```
[the remark this answers is not in the document]
Comment (anna-lach) [reply packages/components/popover/src/popover.stories.ts:154]: This is just an example to show how the dialog can be opened with the popover, but it's opening the dialog with "general/default" styling, not our dialog component. I'm thinking if maybe that story is not really necessary, what do you think about it @jpzwarte ?

[the remark this answers is not in the document]
Comment (jpzwarte) [reply packages/components/popover/src/popover.stories.ts:154]: I agree, that example is not really part of `<sl-popover>`

[the remark this answers is not in the document]
Comment (anna-lach) [reply packages/components/popover/src/popover.stories.ts:154]: I'll remove it :)

**Pull request merged**